### PR TITLE
HotFix: fixed path for closePRForAnIssue action

### LINKED ISSUE #533: bug: `/unassign` is broken
After some recent merges, the `/unassign` command appears to have been broken. It prints an error when you comment it. 

> Error: TypeError: (0 , __1.closePullRequestForAnIssue) is not a function

Here is a go-through showing you how it is behaving right now.  https://github.com/web4er/ubiquibot/issues/37

### PR DESCRIPTION
**Pull request by wannacfuture**
Resolves #533

### CODE REVIEW

File: src/handlers/comment/handlers/unassign.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { removeAssignees } from "../../../helpers";
 import { getBotContext, getLogger } from "../../../bindings";
 import { Payload } from "../../../types";
 import { IssueCommentCommands } from "../commands";
-import { closePullRequestForAnIssue } from "../..";
+import { closePullRequestForAnIssue } from "../../assign/index";
```
Comment (0xcodercrane) [inline src/handlers/comment/handlers/unassign.ts:5]: the changes is different from the original one???

Comment (wannacfuture) [reply src/handlers/comment/handlers/unassign.ts:5]: lol not sure about this
Anyway just wanna have a look if it is fixed on some hunter's local with this pr.

Comment (molecula451) [reply src/handlers/comment/handlers/unassign.ts:5]: yeah it's working

Comment (wannacfuture) [reply src/handlers/comment/handlers/unassign.ts:5]: Cool. @0xcodercrane Seems we can merge this.

**Review by 0xcodercrane — APPROVED** [review]

**Pull request merged**